deepmd/common.py

```diff
@@ -424,6 +424,8 @@ def get_activation_func(
     RuntimeError
         if unknown activation function is specified
     """
+    if activation_fn is None or activation_fn in ['none', 'None']:
+        return None
```

Comment:
You can return `lambda x: x` for None, and this will not be a special case but have the same behavior as other activation functions.